from django.test import TestCase
from efill_app.models import PersonalInfo
from django.core.exceptions import ValidationError


class PersonalInfoTest(TestCase):
	pass
	